libtelemetry/src/main/java/com/mapbox/services/android/telemetry/MapboxTelemetry.java

```diff
@@ -124,6 +136,12 @@ IntentFilter obtainEventReceiverIntentFilter() {
   }
 
   private void initializeTelemetryClient(String accessToken, String userAgent) {
+    if (TextUtils.isEmpty(accessToken) || TextUtils.isEmpty(userAgent)) {
```

Comment:
We're already checking ☝️ (👀 `isTelemetryClientInitialized()`). What about fixing `isTelemetryClientInitialized()` check?